pkgs/by-name/po/podman-desktop/package.nix

```diff
@@ -27,14 +27,14 @@ stdenv.mkDerivation (finalAttrs: {
   passthru.updateScript = _experimental-update-script-combinators.sequence [
     (nix-update-script { })
     (lib.getExe (writeShellApplication {
-      name = "${finalAttrs.pname}-dependencies-updater";
+      name = "podman-desktop-dependencies-updater";
       runtimeInputs = [
         nix
         jq
         gnugrep
       ];
       runtimeEnv = {
-        PNAME = finalAttrs.pname;
+        PNAME = "podman-desktop";
```

Comment:
I intended to use this approach only where it's clearly dependent on `pname`. I don't think it applies to cases like `repo = pname`, which aren't essentially the same. Why should we avoid it then?